lib/chitragupta/util.js

```diff
@@ -10,7 +10,16 @@ function populateServerData(dataParam) {
   data.data.response = {};
 
   const { url } = request;
-  const [endpoint, params] = url.split('?');
+  const indexOfQuestionMark = url.indexOf('?');
```

Comment:
something like `url.split(/\?(.+)/)` works?

Reply:
Don't want to involve regex. Will take a performance hit.